oci/oci.go

```diff
@@ -116,6 +129,23 @@ func (r *Runtime) Name() string {
 // Depending if the container is privileged and/or trusted,
 // this will return either the trusted or untrusted runtime path.
 func (r *Runtime) Path(c *Container) string {
+	if c.runtimeHandler != "" {
+		runtimeHandler, ok := r.runtimes[c.runtimeHandler]
+		if !ok {
```

Comment:
As commented elsewhere, I think this should be an error, rather than falling back to the default.

Reply:
Same here:
>Yes I understand the expectation here, but this will imply the modification of a bunch of things in CRI-O as the current design always expect `func (r *Runtime) Path(c *Container) string` to return something.
@runcom @mrunalp please let me know what you think here, I don't want to introduce too many changes without agreement first.

Reply:
this just affects `Path` so I would just start returning an error here when it makes sense (like here). Long term solution is a complete refactor of all of this but for now let's have a stopgap and return an error